src/components/molecules/collapsible-tree/index.tsx

```diff
@@ -101,7 +101,9 @@ const CollapsibleTreeParent: React.FC<CollapsibleTreeParentProps> = ({
         <div className="flex items-center justify-between">
           <div className="gap-x-small flex items-center">{children}</div>
           <div className="flex items-center gap-x-xsmall">
-            <Actionables customTrigger={Trigger()} actions={actions} />
+            {actions && (
```

Comment:
Guessing that it has nothing to with your PR at all, but seems like the dashed lines between "leafs" are broken, and not extending to the next leaf. Is that also how it looks on your end? 😄

Reply:
![image](https://user-images.githubusercontent.com/45367945/164643792-dada614a-8c64-4d8f-a3da-2d4312becbfc.png)

This is how it looks on my end 🤨  But again not related to your PR, so feel free to merge 😄